server/api/crud/alert_activation.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import mlrun.utils.singleton
+
+valid_partition_intervals = ["DAY", "YEARWEEK", "MONTH"]
+partition_name_format_mapping = {
+    "YEARWEEK": "%Y%V",  # ISO week number (for weeks starting Monday)
```

Comment:
```suggestion
    "YEARWEEK": "%Y%W",  # ISO week number (for weeks starting Monday)
```

Reply:
:+1: I guess my concern is that `%V` is an extended format, not necessarily supported on all platforms and in some cases not documented (as mentioned [here](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes). Given that this is strictly server-side code, I guess that's not really an issue. 

Out of curiosity, is there a specific reason you went with the ISO-8601 implementation for `YEARWEEK` (mode 1)? It supports many other modes (and that is not the default mode).

Reply:
@theSaarco
> Out of curiosity, is there a specific reason you went with the ISO-8601 implementation for YEARWEEK (mode 1)? It supports many other modes (and that is not the default mode).
 
 just because in python it is much easier to work with weeks which start from Monday rather than from Sunday